pkg/csi/service/vanilla/controller.go

```diff
@@ -209,6 +213,19 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 			return err
 		}
 	}
+	// Go module to keep the metrics http server running all the time.
```

Comment:
Can we move this to `func (c *controller) Init`?

Reply:
Its already in the Init method. Am I missing something?

Reply:
the diff was not showing up in the init previously. we are good.